db/migrate/20190901143302_create_friendly_id_slugs.rb

```diff
@@ -0,0 +1,21 @@
+MIGRATION_CLASS =
+  if ActiveRecord::VERSION::MAJOR >= 5
+    ActiveRecord::Migration["#{ActiveRecord::VERSION::MAJOR}.#{ActiveRecord::VERSION::MINOR}"]
+  else
+    ActiveRecord::Migration
+  end
```

Comment:
Why on earth is this needed - we know what version of `ActiveRecord` we are starting from so there is no need for all this noise. I'm guessing it's automatically generated code, but we have no need for it.